SpatialGDK/Source/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -56,15 +56,15 @@ void USpatialPlayerSpawner::SendPlayerSpawnRequest()
 
 void USpatialPlayerSpawner::ReceivePlayerSpawnResponse(Worker_CommandResponseOp& Op)
 {
-	UE_LOG(LogTemp, Warning, TEXT("Got a response"));
+	UE_LOG(LogSpatialPlayerSpawner, Warning, TEXT("Got a response"));
```

Comment:
Should the warning category here be Warning?

Reply:
I believe this log needs updating. Something like TEXT("Received player spawn response %s", op.workerID) or something. Also make it verbose probably.